src/movies.js

```diff
@@ -1,25 +1,175 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
 
+function getAllDirectors(moviesArray) {
+    let allDirectors=moviesArray.map(x=>x.director)
+   return allDirectors
+}
```

Comment:
De todas formas poco a poco os vamos a ir pidiendo esta otra forma!

```suggestion
function getAllDirectors(moviesArray) {
    return moviesArray.map(x=>x.director)
}
```